python_modules/dagster-graphql/dagster_graphql/implementation/fetch_assets.py

```diff
@@ -377,6 +378,24 @@ def get_materialized_partitions_subset(
         )
 
 
+def get_num_failed_partitions(
```

Comment:
prefer refactoring this to return a `PartitionsSubset` of partitions that failed, so we can use this to display ranges of failed partition keys